### PR TITLE
Feat navigation

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Navigation component added 

- Simple navigation

### CODE REVIEW

File: components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+.hy-logo {
+  display: inline-block;
+}
+
+.hy-navbar {
+  display: flex;
+  flex-direction: column;
+  font-size: 1.7rem;
+  max-width: 1200px;
```
Comment (GuptaEesh) [inline components/navigation.css:14]: I think rem would be better here.

Comment (junaidshaikh-js) [reply components/navigation.css:14]: This max-width is for the screen sizes.

**Review by ZuberDunge — COMMENTED** [review]
Looks good to me.

**Review by ZuberDunge — COMMENTED** [review]
Code looks clean and readable. good work junaid.

**Review by GuptaEesh — COMMENTED** [review]
Good work.

**Pull request merged**